client/src/app/template-questionnaire/template-questionnaire.data.ts

```diff
@@ -0,0 +1,14 @@
+export interface QuestionnaireData {
```

Comment:
This doesn't account for the full schema of info specified [here](https://github.com/ebi-ait/ingest-ui/blob/master/client/src/app/template-questionnaire/template-questionnaire-form/template-questionnaire-schema.json)

Also how does this account for the schema changing? (as @ami-day is currently doing)